packages/block-editor/src/components/url-input/style.scss

```diff
@@ -31,6 +31,11 @@ $input-size: 300px;
 		}
 	}
 
+	&.has-border input[type="text"] {
```

Comment:
I'm not sure if this is the standard way to introduce the `.has-button` class. I could use `@at-root` to make it nested correctly, but I didn't notice it being used in Gutenberg.

Could you let me know if I have done this correctly or is there a more standard way to do this, preferably with one of your awesome examples?

Thank you so much ❤️